lib/mrtrix3/dwi2mask/synthstrip.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2008-2023 the MRtrix3 contributors.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Covered Software is provided under this License on an "as is"
+# basis, without warranty of any kind, either expressed, implied, or
+# statutory, including, without limitation, warranties that the
+# Covered Software is free of defects, merchantable, fit for a
+# particular purpose or non-infringing.
+# See the Mozilla Public License v. 2.0 for more details.
+#
+# For more details, see http://www.mrtrix.org/.
+
+import shutil
+import argparse
+import os
+import warnings
+from mrtrix3 import MRtrixError
+from mrtrix3 import app, run
+
+
+DEFAULT_CLEAN_SCALE = 2
```

Comment:
This is unused, as is the `-clean_scale` option. This is a component of the `legacy` algorithm, since it duplicates the existing behaviour of the `dwi2mask` C++ binary on `3.0.x`. But it should be omitted here. 

(Users can nevertheless use this functionality explicitly after the fact using `maskfilter clean` if they choose)